Sources/TuistGenerator/Linter/GraphLinter.swift

```diff
@@ -161,11 +186,11 @@ class GraphLinter: GraphLinting {
             LintableTarget(platform: .iOS, product: .framework),
             LintableTarget(platform: .iOS, product: .staticFramework),
             LintableTarget(platform: .iOS, product: .bundle),
-//            LintableTarget(platform: .iOS, product: .appExtension),
-//            LintableTarget(platform: .iOS, product: .messagesExtension),
-//            LintableTarget(platform: .iOS, product: .stickerPack),
-//            LintableTarget(platform: .watchOS, product: .watch2App),
-//            LintableTarget(platform: .watchOS, product: .watchApp),
+            //            LintableTarget(platform: .iOS, product: .appExtension),
+            //            LintableTarget(platform: .iOS, product: .messagesExtension),
+            //            LintableTarget(platform: .iOS, product: .stickerPack),
+            //            LintableTarget(platform: .watchOS, product: .watch2App),
+            //            LintableTarget(platform: .watchOS, product: .watchApp),
```

Comment:
Just double check you're happy with this new alignment, I think by formatting the file it's indented them by accident.

Reply:
it was swiftformat - will push a revert, hopefully won't get flagged as an unformatted PR 🤐

Reply:
ouch, will have to re-apply those change

<img width="655" alt="Screen Shot 2019-08-03 at 11 55 38 AM" src="https://user-images.githubusercontent.com/11914919/62411002-9eed4980-b5e5-11e9-861f-9024743c5ac3.png">